packages/front-end/pages/settings/index.tsx

```diff
@@ -38,10 +39,17 @@ export type SettingsApiResponse = {
       name: string;
       role: MemberRole;
     }[];
+    freeSeats?: number;
+    discountCode?: string;
     subscription?: {
       id: string;
       qty: number;
       trialEnd: Date;
+      current_period_end: number;
+      cancel_at: number | null;
+      canceled_at: number | null;
+      cancel_at_period_end: boolean;
+      planNickname: string;
```

Comment:
We should share this type definition with the back-end since it's identical.

Reply:
@jdorn Can we walk through this together on Monday? I've been spinning my wheels trying to implement this change. Or, if you look at this before then, do you know of any examples of front-end types that we share with the backend that I can use as a reference? Each time I try to import this type on a backend file I get tons and tons of errors.